src/funcTest/groovy/info/solidsoft/gradle/pitest/functional/AcceptanceTestsInSeparateSubprojectFunctionalSpec.groovy

```diff
@@ -1,16 +1,21 @@
 package info.solidsoft.gradle.pitest.functional
 
+import groovy.transform.CompileDynamic
 import nebula.test.functional.ExecutionResult
 
+@CompileDynamic
 class AcceptanceTestsInSeparateSubprojectFunctionalSpec extends AbstractPitestFunctionalSpec {
 
-    def "should mutate production code in another subproject"() {
+    void "should mutate production code in another subproject"() {
         given:
-            copyResources("testProjects/multiproject", "")
```

Comment:
I was done on purpose. See: https://solidsoft.wordpress.com/2014/05/20/more-compact-automatic-spock-specification-formatting-in-idea-13/

Reply:
Will disable Indentation rule.

Reply:
Unless there is one to have it indented as is in the code :).